website/developer-docs/setup/full-dev-environment.md

```diff
@@ -23,14 +23,20 @@ Depending on your platform, some native dependencies might be required. On macOS
 :::
 
 :::info
-As long as [this issue](https://github.com/xmlsec/python-xmlsec/issues/252) is open, a workaround is required to install a compatible version of `libxmlsec1` with brew, see [this comment](https://github.com/xmlsec/python-xmlsec/issues/254#issuecomment-1511135314).
+As long as [this issue](https://github.com/xmlsec/python-xmlsec/issues/252) about `libxmlsec-1.3.0` is open, a workaround is required to install a compatible version of `libxmlsec1` with brew, see [this comment](https://github.com/xmlsec/python-xmlsec/issues/254#issuecomment-1511135314).
 :::
 
+First, you need to create an isolated Python environment. To create the environment and install dependencies, run the following commands in the same directory as your authentik git repository:
+
 ```shell
 poetry shell # Creates a python virtualenv, and activates it in a new shell
-poetry install # Install all required dependencies, including development dependencies
+make install # Install all required dependencies, including development dependencies
 ```
 
+:::info
+To use a Poetry virtual environment for the first time, you will need to run `poetry init` to create a `pyproject.toml` file, which lists dependencies (such as supported Python versions).
```

Comment:
that's not true, we have the pyproject.toml in the repo which lists everything

Reply:
```suggestion
To use a Poetry virtual environment for the first time, you will need to run `poetry init` to create a `pyproject.toml` file, which lists dependencies (such as supported Python versions).
```
```suggestion
```